go/vt/schemadiff/table.go

```diff
@@ -1359,8 +1473,18 @@ func (c *CreateTableEntity) Apply(diff EntityDiff) (Entity, error) {
 		dupCreateTable.Comments = &d
 	}
 	dup := &CreateTableEntity{CreateTable: *dupCreateTable}
-	if err := dup.apply(alterDiff); err != nil {
-		return nil, err
+	for diff != nil && !diff.IsEmpty() {
```

Comment:
I guess it's explicitly not allowed / supported to supply a list of diffs where there might be an empty diff in that sequence? Because those logic would halt at the first empty diff even if that diff references a next non empty diff.